src/main/java/seedu/stock/model/stock/AccQuantity.java

```diff
@@ -0,0 +1,60 @@
+package seedu.stock.model.stock;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.stock.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a source's accumulated quantity.
+ */
+public class AccQuantity {
```

Comment:
I think you shouldn't use the short form of accumulated here for the class name?

Reply:
I agree with Ashley here.

Reply:
Noted!